.github/workflows/bump-version.yml

```diff
@@ -0,0 +1,35 @@
+name: Bump version
+
+on:
+  push:
+    paths:
+      - version.txt
```

Comment:
This workflow is triggered by pushed any branch and include changed of `version.txt`.